### PR TITLE
Feature/panel

### PR DESCRIPTION
**Pull request by orgilor**

### CODE REVIEW

File: lib/src/panel/panel.service.ts [hunk 1]
```diff
@@ -112,6 +124,17 @@ export class DatoPanel {
     }
   }
 
+  private createBackdrop() {
+    this.backdropElement = this.document.createElement('DIV');
```
[the remark this answers is not in the document]
Comment (orgilor) [reply lib/src/panel/panel.service.ts]: why addClass and not classList.add ?

[the remark this answers is not in the document]
Comment (NetanelBasal) [reply lib/src/panel/panel.service.ts]: we have addClass abstraction in the core. Use it.

File: lib/src/panel/panel.service.ts [hunk 2]
```diff
@@ -23,7 +23,7 @@ export type DatoPanelOptions<E = any> = {
   width?: number | string;
   viewContainerRef?: ViewContainerRef;
   offset?: { top?: number; left?: number };
-  backdrop?: boolean;
+  backdrop?: { enabled: boolean; selector?: string };
```
Comment (NetanelBasal) [inline lib/src/panel/panel.service.ts]: Nice!

**Pull request merged**